components/common/Header.tsx

```diff
@@ -8,7 +8,7 @@ const Header = (props) => (
       <Link href="/">
         <a>
           <object data="/logo.svg" type="image/svg+xml">
-            <img src="/logo.png" alt="Arm-64.com Logo" />
+            <img src="/logo.png" alt="aarch-64.com Logo" />
```

Comment:
aarch64.com 😄

Reply:
Nice spot, not that it was right originally 😂 I'll correct now!